src/software/python_bindings.cpp

```diff
@@ -47,19 +48,30 @@ namespace py = pybind11;
  * Adapted from: https://stackoverflow.com/a/47749076
  *
  * @param m The module to define the sender/receiver in
- * @param The name to insert into the binded class name (ex. {name}ProtoSender)
+ * @param The name to insert into the binded class name (ex. {name}ProtoListener)
  */
 template <typename T>
 void declareThreadedProtoUdpSender(py::module& m, std::string name)
 {
     using Class              = ThreadedProtoUdpSender<T>;
-    std::string pyclass_name = name + "ProtoSender";
+    std::string pyclass_name = name + "ProtoUdpSender";
     py::class_<Class, std::shared_ptr<Class>>(m, pyclass_name.c_str(),
                                               py::buffer_protocol(), py::dynamic_attr())
         .def(py::init<std::string, int, bool>())
         .def("send_proto", &Class::sendProto);
 }
 
+template <typename T>
+void declareThreadedProtoRadioSender(py::module& m, std::string name)
+{
+    using Class              = ThreadedProtoRadioSender<T>;
+    std::string pyclass_name = name + "ProtoRadioSender";
+    py::class_<Class, std::shared_ptr<Class>>(m, pyclass_name.c_str(),
+                                              py::buffer_protocol(), py::dynamic_attr())
+        .def(py::init<>())
+        .def("send_proto", &Class::sendProto);
+}
+
 template <typename T>
 void declareThreadedProtoUdpListener(py::module& m, std::string name)
```

Comment:
Could you add javadocs for these two functions?

Reply:
done